test/resttests.ts

```diff
@@ -38,75 +38,75 @@ describe('Rest Tests', function () {
 
         let restRes: restm.IRestResponse<HttpBinData> = await _rest.get<HttpBinData>('https://httpbin.org/get');
         assert(restRes.statusCode == 200, "statusCode should be 200");
-        assert(restRes.result.url === 'https://httpbin.org/get');
+        assert(restRes.result && restRes.result.url === 'https://httpbin.org/get');
```

Comment:
For all of these, I am not sure it adds value to have the restRes.result check before doing an equality check. It fails either way and since this is just a test I don't think we need it. Makes it harder to read too.

Reply:
Same comment for all places this has been added.

Reply:
I added these checks because I usually compile code with "strict" type checking enabled. If this option is disabled, then it would produces no error message, but in my eye, it is actually semantically wrong.